experimental/HttpBenchmark/HttpBenchmark.csproj

```diff
@@ -10,7 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.CommandLineUtils.Sources" Version="1.2.0-*" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.Extensions.CommandLineUtils.Sources" Version="2.0.0-*" PrivateAssets="All" />
```

Comment:
Thanks, already filed a bug for this, but I wanted to get stuff working again ASAP

https://github.com/aspnet/benchmarks/issues/246